### DISCUSSION TITLE
Making information "author" and "title" queryable

### DISCUSSION THREAD

**Discussion by gcelano** [korpling/ANNIS#831] [General]
Since a label containing the name of an author and the title of a work applies to all tokens of a work, what is the annotation strategy for that? Do I understand correctly that I should simply apply the same label to all tokens in `<feat/>`? (and not, for example, create a chunk annotation file, in that then I cannot search for single tokens having a specific author?)

**Comment by thomaskrause** [comment]
If the annotation applies to all tokens of the work, I would probably attach it to the document as meta annotations. In relANNIS, only documents and (sub-) corpora can have meta annotations. If you can directly produce GraphML for the new ANNIS 4 graphANNIS backend, you could in theory also append to the textual datasource. If you only have one text per document, this should not be necessary. The different formats from where you convert the data to (rel)ANNIS would either directly support meta annotations for documents, or you could add them in Pepper with the `pepper.before.readMeta` general customization property and special metadata files (https://corpus-tools.org/pepper/generalCustomizationProperties.html). Searching for meta annotations is than of course also possible in ANNIS, in version for you would use the `@*` operator (<http://korpling.github.io/ANNIS/4.10/user-guide/aql/annotations.html#searching-for-annotations> and <http://korpling.github.io/ANNIS/4.10/user-guide/aql/operators.html>).

**Reply by gcelano** [reply]
@* works for one piece of metadata but not for two combined (@* author="Plato" @* work="Phaedo"). How to do that?

**Reply by MartinKl** [reply]
Since `author` and `work` are probably data of the same node, your query fragment always has an empty result, as the operator assumes two distinct nodes. The following example query should work:

```
node @* author="Plato" & #1 @* work="Phaedo"
```

Alternatively, if you want to chain, you can do the following:
```
node @* author="Plato" _ident_ work="Phaedo"
```
